jvm/BUILD

```diff
@@ -49,6 +49,11 @@ kt_kotlinc_options(
 
 ktlint_config(
     name = "lint_config",
-    editorconfig = "//:.editorconfig",
+    #    editorconfig = "//:.editorconfig",
```

Comment:
intentional?

Reply:
Yep, will remove the comment in my next push